cloudvolume/cloudvolume.py

```diff
@@ -160,6 +157,27 @@ def __init__(self, cloudpath, mip=0, bounded=True, autocrop=False, fill_missing=
 
     self.pid = os.getpid()
 
+  @classmethod
+  def from_numpy(cls, arr, vol_path='file://tmp/'+generate_random_string(),
+                  resolution=(4,4,40), voxel_offset=(0,0,0), 
+                  chunk_size=(64,64,64)):
+    mkdir(vol_path)
+    if arr.dtype == np.uint8 or arr.dtype == np.float32:
+      layer_type = 'image'
+    elif arr.dtype == np.uint32 or arr.dtype == np.uint64:
+      layer_type = 'segmentation'
+
+    # cloud volume use fortran order, so we need to transpose 
+    # arr = np.transpose(arr)
+    # import pdb; pdb.set_trace()
+    info = cls.create_new_info(1, layer_type, arr.dtype.name, 'raw', resolution, voxel_offset, arr.shape)
+    vol = CloudVolume(vol_path, info=info, bounded=True, autocrop=True) 
```

Comment:
I'm not sure what `autocrop` is buying us here since the size of the volume is exactly equal to the new volume. Maybe we should turn it off?

Reply:
agree.